app/services/ReviewService.scala

```diff
@@ -100,6 +101,34 @@ class ReviewService @Inject() (publishedService: PublishedService, repository: A
     }
   }
 
+  def factCheckComplete(id: String, info: ApprovalProcessStatusChange): Future[RequestOutcome[Unit]] =
+    getContentToUpdate(id, StatusSubmittedForFactCheck) flatMap {
+      case Right(_) =>
+        // Check that all pages have been reviewed
```

Comment:
Think this and the following TODO should be captured in a ticket

Reply:
will create tickets for these - they relate to the stale data which we have to sort out